client/web/src/enterprise/user/routes.tsx

```diff
@@ -33,6 +35,14 @@ export const enterpriseUserAreaRoutes: readonly UserAreaRoute[] = [
     ...userAreaRoutes,
     ...enterpriseNamespaceAreaRoutes,
 
+    // Sourcegraph app specific route (cody/app settings page) this route won't be
+    // available for any non-app deploy types see enterprise userAreaHeaderNavItems
+    // for more context on user settings page
```

Comment:
nit: this comment was hard to parse for me, here's a suggested rewording.

```suggestion
    // Sourcegraph app specific route (cody/app settings page)
    // This route won't be available for any non-app deploy types.
    // See userAreaHeaderNavItems in client/web/src/enterprise/user/navitems.ts
    // for more context on user settings page.
```